### ISSUE TITLE
 the ability to remove the shield from the left hand

### ISSUE THREAD

**Issue by ghost** [Quplet/NoShieldDelay#1]
I play on hypixel version 1.17.1
When blocking with a sword
The shield rises from which the blocking of the sword is reset

Is it possible to add to the config the ability to disable the raising of the shield or make a big delay

type
Shield Blocking= Off
Disabled by defaul
![2022-08-04_23 12 23](https://user-images.githubusercontent.com/80956290/182955577-b8bab14c-ae84-42a4-87a6-9849fb2451d0.png)
![2022-08-04_23 12 27](https://user-images.githubusercontent.com/80956290/182955603-c2550ecc-f1f8-48ef-a958-65f02160af74.png)
![2022-08-04_23 12 34](https://user-images.githubusercontent.com/80956290/182955616-a0ff93f2-67f2-45b4-af7a-d9b5bda94bc0.png)

**Comment by Quplet** [comment]
Yes, you can disable or customize the mod from the config file located (most likely) in `.minecraft/config/no-shield-delay.toml`

Another thing tho, this mod is useless on Hypixel. It only works server side, so even if you have it installed, it will do no good unless the server also has it installed.
